packages/twenty-server/src/engine/twenty-orm/repository/workspace-select-query-builder.ts

```diff
@@ -50,6 +50,48 @@ export class WorkspaceSelectQueryBuilder<
     return super.getMany();
   }
 
+  override getRawOne<U>(): Promise<U | undefined> {
+    this.validatePermissions();
+
+    return super.getRawOne();
+  }
+
+  override getRawMany<U>(): Promise<U[]> {
```

Comment:
```suggestion
  override getRawMany<U = any>(): Promise<U[]> {
```
to avoid having to pass any everywhere anyway (or casting)